packages/cli/src/models/local/LocalController.ts

```diff
@@ -64,10 +64,10 @@ export default class LocalController {
       const path = `${buildFolder}/${file}`;
       if(this.hasBytecode(path)) {
         const contractData = fs.parseJson(path);
-        const isLibrary = contractData.ast.nodes[1].contractKind === "library";
         const isProjectContract = contractData.sourcePath.indexOf(sourceFolder) === 0;
+        const isLibrary = contractData.ast && contractData.ast.nodes.find((node) => node.name === contractData.contractName && node.contractKind === 'library');
```

Comment:
The existence check for contractData.ast may be unnecessary - it was an edge case I ran into in the Truffle tests when working with the contract with bytecode `0x`, that contract has no ast on it. I spent a while trying to compose a contract that would compile into the same an was unsuccessful; a completely blank contract file won't compile at all and even just a pragma declaration will generate some viable bytecode. If there's no real world scenario that leads to this I'd suggest possibly modifying the truffle test and removing this existence check.